### ISSUE TITLE
About the repo location

### ISSUE THREAD

**Issue by chenrui333** [argrath/jnethack-alpha#11]
👋 Looks like `https://scm.osdn.net/gitroot/jnethack` is not getting the latest tag vs this github repo, maybe removing its reference (`https://osdn.jp/projects/jnethack/`) in the repo desc? Thanks!

![image](https://github.com/jnethack/jnethack-alpha/assets/1580956/309455c8-8ba1-4bf2-b3eb-0b562ca5d5af)

Also just curious is there any reason why the repo name carries with `-alpha` in it? 

relates to https://github.com/Homebrew/homebrew-core/pull/153272

**Comment by argrath** [comment]
This repo is for bleeding-edge codes for testing, so I don't think this is suitable for the source of homebrew.
https://scm.osdn.net/gitroot/jnethack/source.git (or its GitHub mirror https://github.com/jnethack/jnethack-release.git) is more suitable.

**Comment by chenrui333** [comment]
aha, I did not know the release repo, I think that is what I am looking for. Looks like the tags there are the same as in here?

**Comment by chenrui333** [comment]
> https://scm.osdn.net/gitroot/jnethack/source.git

this one actually has some access issue.

**Comment by argrath** [comment]
> Looks like the tags there are the same as in here?

Yes (except `latest`, but I think it is not relevant for homebrew).
